
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

arm_controller:
  type: effort_controllers/JointTrajectoryController
  joints: 
    - joint2_to_joint1
    - joint3_to_joint2
    - joint4_to_joint3
    - joint5_to_joint4
    - joint6_to_joint5
    - joint6output_to_joint6
  gains:
    joint2_to_joint1: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    joint3_to_joint2: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    joint4_to_joint3: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    joint5_to_joint4: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    joint6_to_joint5: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    joint6output_to_joint6: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}

gripper_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - grasping_frame_joint
  gains:
    grasping_frame_joint: {p: 100.0, i: 1.0, d: 1.0, i_clamp: 1.0}
    